tests/unit/type/list.tcl

```diff
@@ -599,6 +599,35 @@ foreach {pop} {BLPOP BLMPOP_LEFT} {
       $rd read
     } {}
 
+    test {SWAPDB awakes blocked client} {
+        r flushall
+        r select 1
+        r rpush k hello
+        r select 9
```

Comment:
that select is excessive (for this test), and needed to just to restore the client state for the next state, let's move it to the end and add a comment?

also, maybe it'll be nice to explicitly select 9 on the rd